src/features/components/molecules/deriv-products-hero/products-heading.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react'
+import { Localize } from 'components/localization'
+import Typography from 'features/components/atoms/typography'
+import { TString } from 'types/generics'
+
+const ProductsHeading: FC<{ title: TString[] }> = ({ title }) => {
+    return (
+        <>
+            <Typography.Heading
+                size={'xlarge'}
+                color={'secondary'}
+                weight={'bold'}
+                align={'left'}
+                pt={'4x'}
+                md={{ pt: '8x' }}
+                className="visible-larger-than-tablet"
+            >
+                {title.map((text, index) => (
+                    <>
```

Comment:
Key prop needs to be added here and can remove in line 20
```suggestion
                    <React.Fragment key={index}>
```

Reply:
good catch bro thanks